libflux/c/Makefile

```diff
@@ -0,0 +1,29 @@
+VALGRIND = valgrind
+VALGRIND_ARGS = --leak-check=full --error-exitcode=1
+
+BIN_STATIC = libflux_memory_tester_static
+BIN_DYNAMIC = libflux_memory_tester_dynamic
```

Comment:
I'm missing a lot of context for how this Makefile works. Are these files generated in order to be able to run these tests remotely?

Reply:
So these programs are compiled solely for the purpose of running valgrind. We are basically exercising our rust library and seeing if valgrind reports any leaks or memory usage errors. You can run these locally. I've only done it on linux though. I'm not sure if valgrind works on OS X.